packages/mobile/src/transactions/TransactionReviewScreen.tsx

```diff
@@ -75,11 +77,19 @@ class TransactionReviewScreen extends React.PureComponent<Props> {
     const confirmationProps = this.getConfirmationProps()
 
     return (
-      <ReviewFrame HeaderComponent={this.renderHeader} navigateBack={navigateBack}>
-        {this.renderCard(type, confirmationProps)}
-      </ReviewFrame>
+      <SafeAreaView style={styles.container}>
+        <ReviewFrame HeaderComponent={this.renderHeader} navigateBack={navigateBack}>
+          {this.renderCard(type, confirmationProps)}
+        </ReviewFrame>
+      </SafeAreaView>
     )
   }
 }
 
+const styles = StyleSheet.create({
+  container: {
```

Comment:
Nit: Seeing a few of these, perhaps it could just be a shared style here:
https://github.com/celo-org/celo-monorepo/blob/master/packages/react-components/styles/styles.ts#L7